### ISSUE TITLE
About the difference between val loss and test loss

### ISSUE THREAD

**Issue by SkyKingL** [thuml/TimeXer#11]
Hello! I would like to ask, when I reproduced the effect on the EPF dataset, I found that the val loss and test loss were quite different. Logically, the distribution of the val dataset and the test dataset should be similar, and the loss of the two should not be so different.

![image](https://github.com/user-attachments/assets/2ab10918-eb9a-40f8-b5fc-c3b4ffd9a339)

**Comment by UP-programmer** [comment]
> Hello! I would like to ask, when I reproduced the effect on the EPF dataset, I found that the val loss and test loss were quite different. Logically, the distribution of the val dataset and the test dataset should be similar, and the loss of the two should not be so different.
> 
> ![image](https://private-user-images.githubusercontent.com/95171250/388584229-2ab10918-eb9a-40f8-b5fc-c3b4ffd9a339.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyODA0NjAsIm5iZiI6MTczMjI4MDE2MCwicGF0aCI6Ii85NTE3MTI1MC8zODg1ODQyMjktMmFiMTA5MTgtZWI5YS00MGY4LWI1ZmMtYzNiNGZmZDlhMzM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTIyVDEyNTYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1ZTM5YTg5Nzc0YTkwMzU0MmU1OGM4YWE2NTQ4MzhiZDA2MDZmNjUzNzEyNDU1ZDdkNjIzMGRmZTZmNmNkNDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dVgTAJ9XT0mHsC8GS4IvgGel_0DDpi6lgbo5jIDz4aY)


where is the log?

**Comment by SkyKingL** [comment]
Hello, I included a log-related picture in my question yesterday. If you need more detailed log information, I have attached my log file. It was conducted on a NVIDIA 3090 24GB GPU.
[EPF.log](https://github.com/user-attachments/files/17878179/EPF.log)

**Comment by SkyKingL** [comment]
I included a picture of the log information in my question...

**Comment by UP-programmer** [comment]
> I included a picture of the log information in my question...

Can we have a talk? I have some questions in how to run the code.

**Comment by XiiTang** [comment]
I also noticed the same issue, the loss on the test set is much higher than on the training set. Can you tell me what the reason might be?
